src/assets/env.js

```diff
@@ -1,3 +1,11 @@
+// Define the environment configuration
 window["env"] = {
     customAuth: true,
-    baseUrl :'http://localhost:5000/template/api/v1/'};
\ No newline at end of file
+    baseUrl: 'http://localhost:5000/template/api/v1/',
+    baseUrls: {
```

Comment:
there should only be one baseUrl and it should be string during the deployment devops will change the value according to the environment, we should not add it like this